src/neuroconv/tools/nwb_helpers/_configure_backend.py

```diff
@@ -44,11 +49,21 @@ def configure_backend(
             neurodata_object.set_data_io(
                 dataset_name=dataset_name, data_io_class=data_io_class, data_io_kwargs=data_io_kwargs
             )
+        # Special ndx-events v0.2.0 types
+        elif is_ndx_events_installed and isinstance(neurodata_object, ndx_events.Events):
+            neurodata_object.set_data_io(
+                dataset_name=dataset_name, data_io_class=data_io_class, data_io_kwargs=data_io_kwargs
+            )
+        # But temporarily skipping LabeledEvents
+        elif is_ndx_events_installed and isinstance(neurodata_object, ndx_events.LabeledEvents):
```

Comment:
How comes we are skipping this? They have data and timestamps don't they?

Reply:
Temporarily. Some test fails for some reason I won't get into since this PR is enough on its own

Reply:
In the follow-up that expands this I'd also include the more general Container attempt as well

Just doing minimal thing to get this working for now

Reply:
OK!